onnxruntime/test/providers/trt/trt_basic_test.cc

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/session/inference_session.h"
+#include "test/providers/provider_test_utils.h"
+#include "test/framework/test_utils.h"
+#include "gtest/gtest.h"
+#include "core/providers/trt/trt_execution_provider.h"
+
+using namespace std;
+using namespace ONNX_NAMESPACE;
+using namespace ::onnxruntime::logging;
+
+namespace onnxruntime
+{
+
+namespace test
+{
+void VerifyOutputs(const std::vector<MLValue>& fetches,
+                   const std::vector<int64_t>& expected_dims,
+                   const std::vector<float>& expected_values)
+{
+    ASSERT_EQ(1, fetches.size());
+    auto& rtensor = fetches.front().Get<Tensor>();
+    TensorShape expected_shape(expected_dims);
+    ASSERT_EQ(expected_shape, rtensor.Shape());
+    const std::vector<float> found(rtensor.template Data<float>(), rtensor.template Data<float>() + expected_values.size());
+    ASSERT_EQ(expected_values, found);
+}
+
+
+TEST(TRTExecutionProviderTest, FunctionTest)
```

Comment:
>FunctionTest [](start = 31, length = 12)

Only having one unit tests seems a little sparse for the amount of code added. Can you please check the code coverage for the new files? Should be around 80% or higher.

There's a VS runsettings file that should set things up to measure coverage for onnxruntime code only at onnxruntime\VSCodeCoverage.runsettings. From VS, Test -> Test Settings -> 'Select Test Settings file' to point it at that file. Test -> Analyze Code Coverage.